blocks/header/header.css

```diff
@@ -1,275 +1,329 @@
+#search .search-icon {
+  left: 432px;
+  width: 40px;
+  border-right: 0 solid;
+  border-radius: 4px 0 0 4px;
 }
 
-header nav[aria-expanded="false"] .nav-hamburger-icon,
-header nav[aria-expanded="false"] .nav-hamburger-icon::before,
-header nav[aria-expanded="false"] .nav-hamburger-icon::after {
-  height: 2px;
-  border-radius: 2px;
-  background: currentcolor;
+#search .search-icon img {
+  opacity: .25;
+  padding: 5px 0 0 12px;
 }
 
-header nav[aria-expanded="false"] .nav-hamburger-icon::before {
-  top: -6px;
+#search input[type="text"] {
+  left: 472px;
+  border-left: 0 solid;
+  border-radius: 0 4px 4px 0;
+  width: 256px;
+  color: #555;
+  padding-left: 4px;
 }
 
-header nav[aria-expanded="false"] .nav-hamburger-icon::after {
-  top: 6px;
+input[type="text"]::placeholder {
+  color: #999;
 }
 
-header nav[aria-expanded="true"] .nav-hamburger-icon {
-  height: 22px;
+#search input:focus[type="text"] {
+  border-color: #66afe9;
+  outline: 0;
+  box-shadow: inset 0 1px 1px rgb(102 175 233 / 60%), 0 0 8px rgb(102 175 233 / 60%);
 }
 
-header nav[aria-expanded="true"] .nav-hamburger-icon::before,
-header nav[aria-expanded="true"] .nav-hamburger-icon::after {
-  top: 3px;
-  left: 1px;
-  transform: rotate(45deg);
-  transform-origin: 2px 1px;
-  width: 24px;
-  height: 2px;
-  border-radius: 2px;
+/* chat */
+#chat {
+  position: absolute;
+  top: 4px;
+  right: 183px;
+  cursor: pointer;
 }
 
-header nav[aria-expanded="true"] .nav-hamburger-icon::after {
-  top: unset;
-  bottom: 3px;
-  transform: rotate(-45deg);
+/* nav */
+nav {
+  position: absolute;
+  bottom: 0;
+  left: 260px;
+  user-select: none;
 }
 
-@media (width >= 900px) {
-  header nav .nav-hamburger {
-    display: none;
-    visibility: hidden;
-  }
+nav a { display: block }
+
+nav ul {
+  list-style: none;
+  position: relative;
 }
 
-/* brand */
-header .nav-brand {
-  grid-area: brand;
-  flex-basis: 128px;
-  font-size: var(--heading-font-size-s);
-  font-weight: 700;
-  line-height: 1;
+/* top level */
+nav>div>ul>li {
+  display: inline-flex;
+  flex-direction: column;
+  justify-content: center;
+  cursor: pointer;
+  height: 50px;
+  padding: 0 14px;
+  font-weight: 600;
+  font-size: 1em;
+  color: var(--link-light-gray);
+  text-transform: uppercase;
+  float: left;
 }
 
-header nav .nav-brand img {
-  width: 128px;
-  height: auto;
+nav>div>ul>li:hover,
+nav>div>ul>li.on {
+  background: var(--dark-blue);
+  color: #fff;
 }
 
-/* sections */
-header nav .nav-sections {
-  grid-area: sections;
-  flex: 1 1 auto;
+/* dropdown menus */
+nav>div>ul>li>ul {
   display: none;
-  visibility: hidden;
-  background-color: var(--overlay-color);
+  position: absolute;
+  top: 50px;
+  margin-left: -14px;
+  padding: 4px 0 14px;
+  background: var(--dark-blue);
+  box-shadow: 0 6px 12px rgb(0 0 0 / 17%);
+}
+nav>div>ul>li.on>ul { display: block }
+
+nav>div>ul>li>ul>li {
+  text-transform: none;
+  padding: 6px 0 0;
+  font-size: .85em;
+  line-height: 1.6em;
```

Comment:
1.3em tightens it up like they have.

Reply:
In the spirit of usability I intentionally added a little extra space here.